app/Http/Controllers/SlaReportController.php

```diff
@@ -4,25 +4,61 @@
 
 use App\Conversation;
 use App\User;
+use Carbon\Carbon;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\DB;
 use Exception;
 
 class SlaReportController extends Controller
 {
-    public function slaReport(Request $request)
+    public function slaReport(Request $request,$prev = false, $date_field = 'conversations.created_at', $date_field_to = '')
```

Comment:
Why do all these are arguments

Reply:
Otherwise it gives error
we can also write inside the function